code/game/objects/items/mail.dm

```diff
@@ -0,0 +1,267 @@
+
+/// Mail is tamper-evident and unresealable, postmarked by CentCom for an individual recepient.
+/obj/item/mail
+	name = "mail"
+	gender = NEUTER
+	desc = "An officially postmarked, tamper-evident parcel regulated by CentCom and made of high-quality materials."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail_small"
+	inhand_icon_state = "paper"
+	worn_icon_state = "paper"
+	item_flags = NOBLUDGEON
+	w_class = WEIGHT_CLASS_SMALL
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
+	/// destination tagging for the mail sorter
+	var/sortTag = 0
+	/// who this mail is for and who can open it
+	var/mob/recipient
```

Comment:
This feels like a perfectly acceptable way to get away with mail fraud however, identity theft.
I can still change this if you want, but I just want to verify 👀

Reply:
I think this is probably fine. Unless I'm misunderstanding the system, there's no logical way to where it'd be able to know you switched mobs (in world). This only makes sense.

This does need to be a weakref though to prevent hard deletes.